libnmstate/nm/applier.py

```diff
@@ -262,19 +262,24 @@ def _get_ovs_bridge_port_devices(iface_state):
     OVS bridge.
     Note: Ports must be activated before the ifaces (NM limitation).
     """
-    ifaces = [
-        p[Interface.NAME]
-        for p in iface_state.get(OvsB.CONFIG_SUBTREE, {}).get(
-            OvsB.PORT_SUBTREE, []
-        )
-    ]
-    ports = [ovs.PORT_PROFILE_PREFIX + iface for iface in ifaces]
+    ports = []
+    ifaces = []
+    for port in iface_state.get('bridge', {}).get('port', []):
+        ports.append(ovs.PORT_PROFILE_PREFIX + port['name'])
+        if port['type'] == 'bond':
+            for slave in port.get('link-aggregation', {}).get('slaves', []):
```

Comment:
in case port.get('link-aggregation', {}) returns {} 
(if i understand right, it will return that in case it didnt find anything)
would the get('slaves', []) operates on {} and will assert?

Reply:
If it returns `{}` we call the second `.get('slaves', [])`, where `[]` is the default (in case the key `'slaves'` is not found).

What do you mean by "assert"?
```python
>>> bridge = {}
>>> bridge.get('link-aggregation', {})
{}
>>> bridge.get('link-aggregation', {}).get('slaves', [])
[]
```

Reply:
throwing exception,

ok but it doesn't throw exception as you shown
thanks